packages/safe-apps-sdk/src/safe/index.ts

```diff
@@ -144,6 +146,16 @@ class Safe {
 
     return response.data;
   }
+
+  @requirePermission()
```

Comment:
Is missing because we talked about not complicating things in the first iteration and make a 1:1 system, so I'm  only checking if the user has permissions to call the method. So the permission required for `requestAddressBook` is `requestAddressBook` and we are using the `propertyKey` of the method  itself instead passing parameters